sdk/azidentity/errors.go

```diff
@@ -75,6 +75,23 @@ func (e *AuthenticationFailedError) Error() string {
 		fmt.Fprint(msg, "Response contained no body")
 	}
 	fmt.Fprintln(msg, "\n--------------------------------------------------------------------------------")
+	// adding link to TROUBLESHOOTING.md for relevant errors
```

Comment:
Nitpicking: the code is straightforward enough not to benefit from a comment

Reply:
fixed in commit